src/types/DataTypes.sol

```diff
@@ -58,3 +60,29 @@ library Batch {
         Broker broker;
     }
 }
+
+library MerkleStreamer {
+    /// @notice Struct encapsulating the {SablierV2MerkleStreamerLL} parameters for the
+    /// {SablierV2MerkleStreamerFactory.createMerkleStreamerLL}
+    /// function.
+    /// @param initialAdmin The initial admin of the Merkle streamer contract.
+    /// @param lockupLinear The address of the {SablierV2LockupLinear} contract.
+    /// @param asset The address of the streamed ERC-20 asset.
+    /// @param name The name of the Merkle streamer contract.
+    /// @param merkleRoot The Merkle root of the claim data.
+    /// @param expiration The expiration of the streaming campaign, as a Unix timestamp.
+    /// @param streamDurations The durations for each stream due to the recipient.
+    /// @param cancelable Indicates if each stream will be cancelable.
+    /// @param transferable Indicates if each stream NFT will be transferable.
+    struct CreateWithLockupLinear {
```

Comment:
In future, we will have `createMerkleStreamerLD` and `createMerkleStreamerLT` functions. The constructor parameters of those functions will have a set of different variables than the ones in `createMerkleStreamerLL`.

For example,
1.  `createMerkleStreamerLL` has `ISablierV2LockupLinear lockupLinear` and `LockupLinear.Durations streamDurations`
2. `createMerkleStreamerLD` may have `ISablierV2LockupDynamic lockupDynamic` and `Range range` or `SegmentWithDelta[] segments`
3. `createMerkleStreamerLT` may have `ISablierV2LockupTranched lockupTranched` and `Segment`

That's why I called `CreateWithLockupLinear` instead of giving it a generic name such as `ConstructorParams`.